src/uwtools/resources/platform.jsonschema

```diff
@@ -0,0 +1,29 @@
+{
```

Comment:
I factored out the `platform` schema so that it can be used (and validated) by multiple drivers. The FV3 driver validates both the `fv3` and `platofrm` schema blocks, as it uses both of them.